pkgs/applications/virtualization/colima/default.nix

```diff
@@ -2,8 +2,12 @@
 , buildGo118Module
 , fetchFromGitHub
 , installShellFiles
-, lima
+, lima-unwrapped
 , makeWrapper
+, git
+, perl
+, qemu
+, openssl
```

Comment:
```suggestion
```

`git` can be removed once reverting the `postFetch` removal

---

`perl` doesn't seem used at all

---

`qemu` can be removed if grabbing the `binPath` from lima e.g.

`lima.pasthru.binpath` & `lima.pasthru.unwrapped`

also means colima could support `extraLimaPackages` and do `ourlima = lima.override { extraPackages = extraLimaPackages; }` and get `binpath` from `ourlima.pasthru.binpath`
or people can pass an already overriden `lima` in

---

`openssl` seems to just be for generating a shasum (as part of the `make build`) which we don't ever use, this can be removed once the `Makefile` is no-longer used in the `buildPhase`